README.md

```diff
@@ -1,10 +1,10 @@
 ![ OceanGliders Logo](logo-ocean-gliders.png "OceanGliders Logo")
 
 # OceanGliders format-user-manual
-Here we review issues related to OceanGliders format and vocabularies.
+Here we review issues related to OceanGliders (OG) format and vocabularies.
 A discussion section is also available for any question or comment related to OG format and vocabularies.
 
-You can find the most recent version [here](https://github.com/OceanGlidersCommunity/OG-format-user-manual/blob/main/OG_Format.adoc)
+You can find the most recent version of the OG format [here](https://github.com/OceanGlidersCommunity/OG-format-user-manual/blob/main/OG_Format.adoc)
```

Comment:
A suggestion. The 'here' doesn't add any information. Could be removed.

... of the [OG format](https://...